modules/arm_plugin/src/arm_plugin.cpp

```diff
@@ -96,84 +96,47 @@ InferenceEngine::IExecutableNetworkInternal::Ptr Plugin::LoadExeNetworkImpl(cons
     return std::make_shared<ExecutableNetwork>(transformedModel, cfg, std::static_pointer_cast<Plugin>(shared_from_this()));
 }
 
+bool Plugin::isOperationSupported(const std::shared_ptr<ov::Node>& node, const Converter& converter) const {
+    auto itConversion = converter._conversions.find(node->get_type_info());
+    bool nodeIsSupported = false;
+    if (itConversion != converter._conversions.end()) {
+        if (ngraph::op::is_constant(node) || ngraph::op::is_parameter(node) || ngraph::op::is_output(node)) {
+            nodeIsSupported = true;
+        } else {
+            Converter::Conversion::Ptr layer;
+            try {
+                layer = converter._conversions.at(node->get_type_info())(*node);
+            } catch(...) {
+                nodeIsSupported = false;
+            }
+            if (layer != nullptr) {
+                nodeIsSupported = static_cast<bool>(layer->Validate());
+            }
+        }
+    }
+    return nodeIsSupported;
+}
+
 QueryNetworkResult Plugin::QueryNetwork(const CNNNetwork& network, const ConfigMap& config) const {
     QueryNetworkResult res;
     Configuration cfg{config, _cfg, false};
     auto model = network.getFunction();
     if (model == nullptr) {
          IE_THROW() << "Arm Plugin supports only ngraph cnn network representation";
     }
-    std::unordered_set<std::string> originalOps;
-    for (auto&& node : model->get_ops()) {
-        originalOps.emplace(node->get_friendly_name());
-    }
     auto transformedModel = Transform(model, cfg);
```

Comment:
Why don't move it inside transformation callback?

Reply:
our function `isOperationSupported` is required converter with transformed model that why this transformation callback outside and some tests become is failed